### ISSUE TITLE
Android: Add Scroll bar view

### ISSUE THREAD

**Issue by shuvashish76** [samuelclay/NewsBlur#1870]
Currently we've only a viewbar for articles which lets you know where the current screen is in relation to the maximum height.
Adding a scroll button would allow users to jump screen quickly.

Feature request to add it for Main page, Feeds page and article Reading Page.
Have a look on this [fastscroll](https://github.com/zhanghai/AndroidFastScroll) library.

**Comment by shuvashish76** [comment]
https://github.com/nanihadesuka/LazyColumnScrollbar
Android Jetpack Compose LazyColumn scrollbar implementation

Also see: https://github.com/reddit/IndicatorFastScroll popular library by _reddit_ but looks abandoned

**Comment by samuelclay** [comment]
Yeah all of those libraries look abandoned. I'm not sure we'd benefit from this. Isn't there a default scroll bar already? On iOS, web, and macOS there is one. It's kind of up to Android to have this available by default.
